lib/myfinance/version.rb

```diff
@@ -1,3 +1,3 @@
 module Myfinance
-  VERSION = "1.6.4".freeze
+  VERSION = "1.6.5".freeze
```

Comment:
`VERSION` change shouldn't be in the pull request. This will be done when releasing the new version.